5.5/Dockerfile

```diff
@@ -20,10 +20,23 @@ LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database ser
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
+# Support for atomic run/install/uninstall
+LABEL INSTALL="docker run -t -i --rm --privileged -u 0:0 -v /:/host --net=host \
```

Comment:
Again, I don't disagree, but you can say that about RPM too and we are all using it to install SW as root. Anyway, let's stop here, since it's starting to be unconstructive now I think. Neither me or you can change the way how Atomic is working *right* now. Let's wait for someone from Atomic instead :).